### PR TITLE
Make hipcc part of install prereqs and cleanup commented line

### PR DESCRIPTION
**Pull request by TedThemistokleous**

### CODE REVIEW

File: hip-clang.docker [hunk 1]
```diff
@@ -45,9 +44,6 @@ RUN ldconfig
 # Workaround broken miopen cmake files
 RUN sed -i 's,;/usr/lib/x86_64-linux-gnu/librt.so,,g' /opt/rocm/lib/cmake/miopen/miopen-targets.cmake
 
-# Workaround for broken hip-clang
-# RUN sed -i -e '/if(WIN32)/, /endif()/d' /opt/rocm/lib/cmake/hip/hip-config.cmake && sed -i -n 5p /opt/rocm/lib/cmake/hip/hip-config.cmake
```
Comment (pfultz2) [inline hip-clang.docker:49]: I think the workaround can be removed from sles as well.

File: tools/install_prereqs.sh [hunk 2]
```diff
@@ -60,6 +60,7 @@ else
     libgfortran5 \
     hipblas-dev \
     hipblaslt-dev \
+    hipcc \
```
Comment (pfultz2) [inline tools/install_prereqs.sh:63]: There should probably be a comment that this is a workaround for broken hip-dev package.

**Review by pfultz2 — APPROVED** [review]

**Review by ahsan-ca — APPROVED** [review]

File: hip-clang.docker (hunk 1 above)
Comment (TedThemistokleous) [reply hip-clang.docker:49]: removed

File: tools/install_prereqs.sh (hunk 2 above)
Comment (TedThemistokleous) [reply tools/install_prereqs.sh:64]: Added above

**Pull request merged**